### ISSUE TITLE
createDefaultTransformationRules not working as expected

### ISSUE THREAD

**Issue by cowwoc** [jk1/Gradle-License-Report#133]
I tried invoking `filters = new LicenseBundleNormalizer(bundlePath: "$projectDir/scripts/gradle/license-normalizer-bundle.json", createDefaultTransformationRules: true)` but the default transformation rules for `apache2` aren't being applied. If I omit my custom confirmation, the rule gets used just fine.

My custom rules are:

```
{
  "bundles": [
    {
      "bundleName": "cddl1.1",
      "licenseName": "COMMON DEVELOPMENT AND DISTRIBUTION LICENSE (CDDL) Version 1.1",
      "licenseUrl": "https://oss.oracle.com/licenses/CDDL-1.1"
    }
  ],
  "transformationRules": [
    {
      "bundleName": "cddl1.1",
      "licenseUrlPattern": ".*/CDDL\\+GPL_1_1\\.html"
    }
  ]
}
```

Any ideas?

**Comment by guenhter** [comment]
This is unfortunate: The problem is, that there are two sources for default rules:

1) The `default-license-normalizer-bundle.json` file inside the plugin, which contains a few useful rulses
2) Default rules created out of the given bundles as exact matching rules (created at runtime)

Unfortunately, the flag `createDefaultTransformationRules` only controls the second type of default rules. The `default-license-normalizer-bundle.json` file is used as a fallback when no bundle is defined.

**Comment by guenhter** [comment]
We should think about changing this behavior: Maybe we should create all the default rulses 1) and 2) depending on the given flag....

**Comment by cowwoc** [comment]
Yes please.

**Comment by guenhter** [comment]
Fixed with #135
